config/application.rb

```diff
@@ -61,7 +62,8 @@ class Application < Rails::Application
       mailchimp.js
       main.js
       jsxc.js
-      mobile.js
+      mobile/mobile.js
+      mobile/mobile_file_uploader.js
```

Comment:
Only list files here that you need to have available for `javascript_include_tag` and the like, everything else will get pulled in via the sprockets requires.